README.md

```diff
@@ -251,6 +199,14 @@ See [RELEASES](https://github.com/tallguyjenks/Obsidian-For-Business/releases)
 
 See [CODE OF CONDUCT](CODE_OF_CONDUCT.md)
 
+### Contributors
+
+[Return To Top](#table-of-contents)
+
+None yet! But PR's welcome!
```

Comment:
should edit this to add yourself :)

`[@username][1]` 

and at bottom of document

[1]: link to your github